src/ado2gh/Commands/WaitForMigrationCommand.cs

```diff
@@ -60,6 +60,7 @@ public async Task Invoke(string migrationId, string githubPat = null, bool verbo
                 switch (state)
                 {
                     case RepositoryMigrationStatus.Failed:
+                    case RepositoryMigrationStatus.FailedValidation:
```

Comment:
Can we update this to use the IsFailed/IsPending methods, and while we're at it we should explicitly check for IsPending instead of using default like we are here. Default should throw an exception (NOT an OctoshiftCliException) saying it's an unexpected state.

Reply:
Since `IsSucceeded` is true for succeeded states, `IsPending` is true for pending states, and `IsFailed` is true for both failed and unexpected states, wouldn't the state always trip one of those three? Or does your suggestion posit a future where maybe we make `IsFailed` a bit stricter?

Reply:
My bad, I didn't look close enough at the implementation of IsFailed